jax/numpy/__init__.py

```diff
@@ -22,9 +22,6 @@
 
 from jax._src.numpy.lax_numpy import (
     ComplexWarning as ComplexWarning,
```

Comment:
Leave these in, and add a TODO.

Reply:
Sure! Added TODO with numpy's version that will ship this change.